test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -509,12 +510,15 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 					Group("test-group").
 					GroupTotalCount("2").
 					Queue("test-queue").
+					Label("dontCopyKey", "dontCopyValue").
 					Obj()
 				pod2 := testingpod.MakePod("test-pod2", ns.Name).
 					Group("test-group").
 					GroupTotalCount("2").
 					Request(corev1.ResourceCPU, "1").
 					Queue("test-queue").
+					Label("toCopyKey", "toCopyValue").
+					Label("dontCopyKey", "dontCopyValue").
```

Comment:
nit
```suggestion
					Label("dontCopyKey", "dontCopyAnotherValue").
```

To make sure different values here don't prevent the Workload creation.

Reply:
Done